.github/contributing.md

```diff
@@ -15,7 +15,8 @@ This workflow guides contributors on how to address **[confirmed]** issues in al
 ---
 
 ## Valid Issues
-Only issues with the label **[confirmed]** should be picked up and fixed. If you are interested in a new feature or bug fix, please check if there is already an issue created. If not, create a new issue, and wait for it to be labeled **[confirmed]** by maintainers before proceeding.
+Only issues with the label **[confirmed]** should be picked up and fixed. If you are interested in a new feature or bug fix, please check if there is already an issue created. If not, create a new issue.
```

Comment:
Kind of double now. Perhaps phrase it as follows: 
Only issues with the label **[confirmed]** are guaranteed to be merged. If you have a suggestion for a new feature or bug fix, please check whether it is a duplicate before creating an issue. After issue creation a maintainer will determine whether it gets the **[confirmed]** label.